src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php

```diff
@@ -98,7 +98,7 @@ protected function getCapabilitiesNode()
                 ->scalarNode('platform')->defaultValue('ANY')->end()
                 ->scalarNode('browserVersion')->defaultValue('9')->end()
                 ->scalarNode('browser')->defaultValue('firefox')->end()
-                ->scalarNode('ignoreZoomSetting')->defaultValue('false')->end()
+                ->booleanNode('ignoreZoomSetting')->defaultFalse()->end()
```

Comment:
Yes. Actually, in yaml, `true` and `false` are transformed into booleans, even these are strings.

Adding the capabilities manually actually also fixes the problem, but not the source.
This PR actually fixes the issue which lies within the default value provided by the factory.